reporting/impl/src/main/java/org/jboss/windup/reporting/export/ExportCSVFileRuleProvider.java

```diff
@@ -60,7 +62,8 @@ public Configuration getConfiguration(GraphContext context)
             .addRule()
             .when(Query.fromType(WindupConfigurationModel.class).withProperty(WindupConfigurationModel.CSV_MODE,true)
                         .and(Query.fromType(ClassificationModel.class).as("classifications")
-                                    .and(Query.fromType(InlineHintModel.class).as("hints"))))
+                                    .and(Query.fromType(InlineHintModel.class).as("hints")
```

Comment:
I would like to see this moved into the operation itself. Rewrite does not do short-circuit evaluation, so these queries run even when export is not enabled. In the case of a large application, this is adding quite a bit of time to the windup run.